From 5956d0a70600fe3f70282419c3328e68e8b68f1a Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Sun, 2 Mar 2014 17:44:29 +0100
Subject: [PATCH] gbm: make 'devices' array static

It's only used in this one file, and exporting a symbol named 'devices'
from a shared library is terribly wrong.
---
 src/gbm/main/gbm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gbm/main/gbm.c b/src/gbm/main/gbm.c
index 72eeabf..04cd90c 100644
--- a/src/gbm/main/gbm.c
+++ b/src/gbm/main/gbm.c
@@ -43,7 +43,7 @@
 
 #define ARRAY_SIZE(a) (sizeof(a)/sizeof((a)[0]))
 
-struct gbm_device *devices[16];
+static struct gbm_device *devices[16];
 
 static int device_num = 0;
 
-- 
1.9.0

